flatdata-cpp/include/flatdata/ExternalVector.h

```diff
@@ -83,7 +83,7 @@ template < typename T >
 typename ExternalVector< T >::ValueType
 ExternalVector< T >::grow( )
 {
-    if ( m_data.size( ) > 1024 * 1024 * 32 )
+    if ( m_data.size( ) * sizeof( T ) > 1024 * 1024 * 32 )
```

Comment:
sizeof( T ) == sizeof( void* ) since all T's are just handles.

What you would need is T::size_in_bytes()... or just m_data.size_in_bytes()

Reply:
I guess you should also fix: https://github.com/heremaps/flatdata/blob/master/flatdata-rs/lib/src/vector.rs#L332